docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/RegressionSamplesTemplate.ttinclude

```diff
@@ -9,6 +9,9 @@ using Microsoft.ML.Trainers;
 
 namespace Samples.Dynamic.Trainers.Regression
 {
+<# if (ClassHeader != null) { #>
```

Comment:
>ClassHeader [](start = 7, length = 11)

It used to be on top of method, not the class.
I honestly have no idea which is better, and which is right, and considering we have one class to one method ration anyway, I doubt it's even important and matter.
Just want to point on this one.

Should it be on top of class or method? @shmoradi

Reply:
Will follow previous convention.

---
In reply to: [275453678](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275453678) [](ancestors = 275453678)